server/src/test/java/org/elasticsearch/action/search/ExpandSearchPhaseTests.java

```diff
@@ -117,31 +117,34 @@ void sendExecuteMultiSearch(MultiSearchRequest request, SearchTask task, ActionL
 
                 SearchHit hit = new SearchHit(1, "ID");
                 hit.setDocumentField("someField", new DocumentField("someField", Collections.singletonList(collapseValue)));
-                SearchHits hits = new SearchHits(new SearchHit[] { hit }, new TotalHits(1, TotalHits.Relation.EQUAL_TO), 1.0F);
-                try {
-                    ExpandSearchPhase phase = newExpandSearchPhase(
-                        mockSearchPhaseContext,
-                        new SearchResponseSections(hits, null, null, false, null, null, 1),
-                        null
-                    );
-
-                    phase.run();
-                    mockSearchPhaseContext.assertNoFailure();
-                    SearchResponse theResponse = mockSearchPhaseContext.searchResponse.get();
-                    assertNotNull(theResponse);
-                    assertEquals(numInnerHits, theResponse.getHits().getHits()[0].getInnerHits().size());
+                ExpandSearchPhase phase = newExpandSearchPhase(
```

Comment:
This stuff is what blows up the LoC changed in this PR but I felt it best to inline these values still since their lifecycle is now bound to the `SearchResponseSections`.